2_generate_manifest.py

```diff
@@ -48,7 +53,6 @@ def construct_left_right_hive_partition_dirs(rec):
                 dirs.append(side + '_' + col + '=' + str(rec.get(side + '_' + ocol, None)))
         return os.path.join(*dirs)
 
-    manifest = []
```

Comment:
I'm fine with it now, but a general tip: only change things related to the PR. I understand that you removed this as it is not used anymore, but such things can become confusing quickly, for example if you are trying to understand why certain things were changed.

Reply:
Ok, will revert it